lib/galaxy/config/__init__.py

```diff
@@ -587,7 +586,7 @@ def __init__(self, **kwargs):
             self.nginx_upload_store = os.path.abspath(self.nginx_upload_store)
         self.object_store = kwargs.get('object_store', 'disk')
         self.object_store_check_old_style = string_as_bool(kwargs.get('object_store_check_old_style', False))
-        self.object_store_cache_path = resolve_path(kwargs.get("object_store_cache_path", "database/object_store_cache"), self.data_dir)
+        self.object_store_cache_path = resolve_path(kwargs.get("object_store_cache_path", "object_store_cache"), self.data_dir)
```

Comment:
Wouldn't this break setups where in `galaxy.yml` this property is configured to a path relative to the Galaxy root?